x-pack/plugins/fleet/server/routes/agent_policy/index.ts

```diff
@@ -42,6 +42,7 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
     {
       path: AGENT_POLICY_API_ROUTES.LIST_PATTERN,
       validate: GetAgentPoliciesRequestSchema,
+      options: { access: 'public' },
```

Comment:
are APIs internal by default?

Reply:
Hey Julia! They are since we merged https://github.com/elastic/kibana/pull/161672

We intend to use this to require setting a specific header for all "internal" routes when the `server.restrictInternalApis` setting is `true`. Right now only intended for use by us on serverless.